### PR TITLE
Added enum for exit codes

### PR DESCRIPTION
**Pull request by ErikZilber**
## 📝 Description

Added enum for exit codes and modified existing sys.exit calls to use it

### CODE REVIEW

File: linodecli/exit_codes.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+"""
+This is an enumeration of the various exit codes in Linode CLI
+"""
+
+from enum import Enum
+
+class ExitCodes(Enum):
```
Comment (lgarber-akamai) [inline linodecli/exit_codes.py:7]: We can use the `IntEnum` base class here to avoid needing to get the underlying value of an enum using `.value` 👍

Comment (ErikZilber) [reply linodecli/exit_codes.py]: Fixed!

**Review by lgarber-akamai — APPROVED** [review]
LGTM after linting is fixed, nice work! During the next release we should communicate that some exit codes may have changed since the previous release.

**Review by yec-akamai — APPROVED** [review]
LGTM

**Pull request merged**